fud/fud/stages/xilinx/xclbin.py

```diff
@@ -200,14 +205,14 @@ def read_file(
         compile_xclbin(client, tmpdir)
 
         if self.remote_exec.use_ssh:
-            xclbin = self.remote_exec.close_and_get(
+            return self.remote_exec.close_and_get(
                 client,
                 tmpdir,
                 "xclbin/kernel.xclbin",
+                keep=self.save_temps,
             )
         else:
```

Comment:
Super nit: `else` not really needed here, but understand some people like the symmetry.